proxy/http/HttpTransact.cc

```diff
@@ -542,34 +542,18 @@ HttpTransact::Forbidden(State *s)
 void
 HttpTransact::HandleBlindTunnel(State *s)
 {
-  NetVConnection *vc         = s->state_machine->ua_session->get_netvc();
-  bool inbound_transparent_p = vc->get_is_transparent();
```

Comment:
There is specific handling here for transparent connections, but I don't see that in the replacement code. Is it no longer needed? If so, why not?

Reply:
This flag never had any use. Line 596: `if (!(inbound_transparent_p || url_remap_success) || remap_redirect != nullptr)` , url_remap_process is always true , which makes this flag pointless. Again remap_redirect is set to nullptr in 587. Hence the whole if condition is always false.